docker-compose.yml

```diff
@@ -7,7 +7,7 @@ x-logging: &logging
 
 services:
   l2_execution_engine:
-    image: gcr.io/evmchain/taiko-geth:jolnir
+    image: gcr.dockerproxy.com/evmchain/taiko-geth:katla
```

Comment:
Is this dockerproxy really secure though? Can't find much info about it and seems like an easy way to introduce risks, maybe better to suggest as an alternative to people that can't connect to gcr.io